### ISSUE TITLE
Is it possible to get residual errors?

### ISSUE THREAD

**Issue by kfeng19** [chaaland/EllipseFit.jl#7]
Just wondering if it's possible to get the residual errors of the fitting so that I could eliminate some of the "bad" data points.

**Comment by chaaland** [comment]
Yeah don't see a reason it couldn't be returned by fit. Would you like to try your hand at a PR? Could give a try today if not.

**Comment by kfeng19** [comment]
Didn't have a chance to look into this yet ... I'm not very familiar with all the methods in the code. I can do a little research on how to do it later but feel free to give a try.
BTW, when you refer to "nonlinear least squares", did you mean to use "NLLS" instead of "NNLS"?

**Comment by chaaland** [comment]
On second thought, I think this might be better as a helper method. For example you could write something like 
```julia
function fitted_ellipse_residuals(ellipse, data)
  N, n = size(data)
  A = hcat(x .^ 2, x .* y, y .^ 2, x, y, ones(N))
  x_sol = vec([
     ellipse.conicform.A, 
     ellipse.conicform.B, 
     ellipse.conicform.C, 
     ellipse.conicform.D, 
     ellipse.conicform.E, 
     ellipse.conicform.F,
   ])
  resid = A * x_sol
  return resid
end
```

Returning the residual would likely mean different things for least squares and NLLS so might be better just to have your function like this?

**Comment by kfeng19** [comment]
Just saw you added the F parameter to the conic form which is very timely. Made PR #13 for this function.
